Remove the option to import keys

nicodh opened the pull request:
~~Draft while waiting for https://github.com/deltachat/deltachat-pages/issues/1069~~

Since the other clients just release the removed option we will do the same and hopefully get an updated text soon.

closes #4783

WofWca reviewed (commented):
https://github.com/deltachat/deltachat-android/pull/3594 also removes "prefer E2E encryption". Don't we want to do the same? It also removed "export keys", but for a different reason.
Don't we want to do the same?

Code-wise this looks good, at the current state.

nicodh commented on CHANGELOG.md:
```diff
@@ -12,6 +12,7 @@
 ### Changed
 - tauri: replace `tauri-plugin-shell` with `tauri-plugin-opener` #4699
 - new button styles #4741
+- removed the option to import encryption keys #4783
```
Well I think we should not be strict on that, otherwise you always have to make an additional commit to set the MR number.

nicodh commented:
> [deltachat/deltachat-android#3594](https://github.com/deltachat/deltachat-android/pull/3594) also removes "prefer E2E encryption". It also removed "export keys", but for a different reason. Don't we want to do the same?

We already removed prefer e2e encryption in https://github.com/deltachat/deltachat-desktop/pull/4785

We will keep the export key to enable other mail clients to import them afaik

WofWca reviewed (approved):
If that's the case, then the MR does the job.
